lib/colors.ts

```diff
@@ -160,6 +162,26 @@ export function toPercent(value: number): string {
   return `${clean(100 * value)}%`
 }
 
+function isHexNotation(value: string): boolean {
+  return /^#?([\da-f]{3}|[\da-f]{4}|[\da-f]{6}|[\da-f]{8})$/i.test(value)
+}
+
+const OUTPUT_FORMAT_MAP: Partial<Record<Mode, OutputFormats>> = {
+  hsl: 'hsl',
+  lab: 'lab',
+  lch: 'lch',
+  oklab: 'oklab',
+  p3: 'p3',
+  rgb: 'rgb'
+}
+
+export function getOutputFormat(value: string): OutputFormats | undefined {
```

Comment:
It is not clear that `value` is mode. Let’s rename function to `getOutputFormat(color: string)`

Reply:
Removed it to `view/code/index.ts` due to cyclic dependency between `lib/colors.ts` and `stores/formats.ts`